### PR TITLE
Kaleidoscope 2.4

### PR DESCRIPTION
**Pull request by florian-muc**
Updating Kaleidoscope and ksdiff to the most recent release.

After making all changes to a cask, verify:

- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [X] `brew audit --cask {{cask_file}}` is error-free.
- [X] `brew style --fix {{cask_file}}` reports no offenses.

### CODE REVIEW

File: Casks/kaleidoscope.rb [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+  sha256 "f3cb0d9135fecb430c949bb6d2e53eacb6f7b81873b707fa021bd6ae1c16b1b9"
 
-  url "https://updates.kaleidoscope.app/v2/prod/Kaleidoscope-#{version.before_comma}-build-#{version.after_comma}.app.zip",
-      verified: "updates.kaleidoscope.app/"
+  url "https://updates.kaleidoscope.app/v2/prod/Kaleidoscope-#{version.before_comma}-#{version.after_comma}.app.zip"
   appcast "https://updates.kaleidoscope.app/v2/prod/appcast"
   name "Kaleidoscope"
-  desc "Spots differences in text and image files"
-  homepage "https://www.kaleidoscopeapp.com/"
+  desc "Spots and merge differences in text and image files or folders"
```
Comment (vitorgalvao) [inline Casks/kaleidoscope.rb:8]:
```suggestion
  desc "Spot and merge differences in text and image files or folders"
```

File: Casks/ksdiff.rb [hunk 2]
```diff
@@ -1,9 +1,10 @@
 cask "ksdiff" do
-  version "2.3.6,123-jan-18-2021"
-  sha256 "6693effaf5132190229877db7cec00a9d16f2f1ecfe755c81cfe8c4a2c748b38"
+  version "2.4,124-feb-4-2021"
+  sha256 "cedb705806f28272f319c520d320a6c716ff57f675bdd03c73432dc5a0184eef"
 
-  url "https://updates.kaleidoscope.app/v2/prod/ksdiff-#{version.before_comma}-build-#{version.after_comma}.zip"
+  url "https://updates.kaleidoscope.app/v2/prod/ksdiff-#{version.before_comma}-#{version.after_comma}.zip"
   name "ksdiff"
+  desc "Command-line tool for App Store version Kaleidoscope"
```
Comment (vitorgalvao) [inline Casks/ksdiff.rb:7]:
```suggestion
  desc "Command-line tool for the App Store version of Kaleidoscope"
```

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/kaleidoscope.rb (hunk 1 above)
Comment (florian-muc) [reply Casks/kaleidoscope.rb]: Thanks, will update.

File: Casks/ksdiff.rb (hunk 2 above)
Comment (florian-muc) [reply Casks/ksdiff.rb]: Thanks, will update.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**